libs/lmdb/Makefile

```diff
@@ -0,0 +1,98 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lmdb
+PKG_VERSION:=0.9.23
+PKG_RELEASE:=1
+
+PKG_SOURCE:=LMDB_$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/LMDB/lmdb/tar.gz/LMDB_$(PKG_VERSION)?
+PKG_HASH:=abf42e91f046787ed642d9eb21812a5c473f3ba5854124484d16eadbe0aa9c81
+PKG_BUILD_DIR:=$(BUILD_DIR)/lmdb-LMDB_$(PKG_VERSION)
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=OLDAP-2.8
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+MAKE_PATH:=libraries/liblmdb
+
+define Package/lmdb/Default
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Lightning Memory-Mapped Database
+  URL:=https://symas.com/lmdb/
+endef
+
+define Package/lmdb
+	$(call Package/lmdb/Default)
+  TITLE+= shared library
+endef
+
+define Package/lmdb/description
+  LMDB is an ultra-fast, ultra-compact key-value
+  embedded data store developed for the OpenLDAP Project.
+endef
+
+define Package/lmdb-utils
+	$(call Package/lmdb/Default)
+  TITLE+= utils
+  DEPENDS+= lmdb
+endef
+
+define Package/lmdb-utils/description
+  LMDB environment status and copy tool
+endef
+
+define Package/lmdb-test
+	$(call Package/lmdb/Default)
+  TITLE+= test
+  DEPENDS+= lmdb
+endef
+
+define Package/lmdb-test/description
+  LMDB test application
+endef
+
+TARGET_CFLAGS += $(FPIC)
```

Comment:
The -fPIC flag is [hard-coded](https://github.com/LMDB/lmdb/blob/332718f1487d5a08324b522716e2cc3915b5ffea/libraries/liblmdb/Makefile#L90) after `$(CFLAGS)`.  IIRC, this voids anything we're doing here.  I would patch that `Makefile` to add
`FPIC?=-fPIC`, and then use `$(FPIC)` instead of `-fPIC` in the recipe lines.  Then this line can be removed, and we gain a bit of efficiency with the static library built without PIC.
An alternative that does not require patching would be to add `$(FPIC)` to `CPPFLAGS`, but it would compile everything with PIC.

Reply:
fPIC param removed. I will push change upstream so we can hopefully get rid of that patch in the future.

Reply:
Good idea.  Do you plan on including `liblmdb.pc`?  They won't need to our `PKG_VERSION` trick, and if they keep it there, we won't have to maintain it here.

Reply:
Yes